src/devices/Mcp3428/Mcp3428Async.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Mcp3428
+{
+    using static Helpers;
```

Comment:
We usually try to avoid the using static directive so that it is more obvious when reading the code where do static methods come from, so please remove this.